doc/source/operations.rst

```diff
@@ -55,23 +55,60 @@ Once the appropriate profile has been selected, it can be applied by running the
 Updates
 =======
 
-Update Airship UCP Services
----------------------------
+SUSE Containerized OpenStack is delivered as an rpm package, so performing updates is generally accomplished by simply updating the rpm package to the latest version and redeploying the cloud using the steps outlined in the installation procedure. This is the typical update path and will incorporate all recent changes, as well as automatically updating component chart and image versions. However, there may be use cases where it is desirable to update services or components to specific versions. 
+
+Updating OpenStack Version
+--------------------------
+
+To make a global change to the OpenStack version used by all component images, create a key in ${WORKSPACE}/env/extravars called "suse_osh_image_version" and set it to the desired value. For example, to use the "ocata" version, add the following line to the extravars file:
+
+.. code-block:: yaml
+
+   suse_osh_image_version: "ocata"
```

Comment:
the default is already rocky for suse_opestack_image_version

Reply:
This was just an example on how and where to change the value, but I can use a newer version in the example.